Controller/ChangePasswordController.php

```diff
@@ -86,6 +86,15 @@ public function changePasswordAction(Request $request)
 
         return $this->render('@FOSUser/ChangePassword/change_password.html.twig', array(
             'form' => $form->createView(),
+            'data' => $this->setUserData(),
         ));
     }
+
+    /*
+     * Function to override to send data to the template
```

Comment:
In french we call it 'Fonction', I have translated it to 'function' but you're right in english it's a 'Method'